.pending/improvements/sdk/4349-autogen-alias-f

```diff
@@ -0,0 +1 @@
+#4349 autogen alias files
```

Comment:
No need for a pending log entry here I think.

Reply:
I agree